app/controller/button/DigitizeButtonController.js

```diff
@@ -275,6 +354,17 @@ Ext.define('CpsiMapview.controller.button.DigitizeButtonController', {
                 geometry3857: Ext.JSON.decode(geoJson).geometry
             };
         }
+        else if (type === 'Circle') {
+            // ol circle objects consist of a center corrdinate and a radius in the
```

Comment:
Typo -> `coordinate`

Reply:
solved in https://github.com/compassinformatics/cpsi-mapview/pull/177/commits/b79cde00e7b4d84cc737d59866c7570cad768ee5